README.md

```diff
@@ -1,2 +1,145 @@
-# hello_microverse_project
-The main goal of this project is to master how to use git and GitHub and other tools learned earlier. 
+ <a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
```

Comment:
- [ ] These two links on the Table of Contents are broken. Kindly remove them if you have no content for their sections. We should avoid having broken links in our documentation as it looks less professional.